src/constants.ts

```diff
@@ -73,18 +76,8 @@ export const REACTOR_ADDRESS_MAPPING: ReactorMapping = {
     [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
     [OrderType.Relay]: "0x0000000000A4e21E2597DCac987455c48b12edBF",
   },
-};
-
-export const REACTOR_CONTRACT_MAPPING: ReactorMapping = {
-  ...constructSameAddressMap({
-    [OrderType.Dutch]: "0x6000da47483062A0D734Ba3dc7576Ce6A0B645C4",
-    [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
-    [OrderType.Relay]: "0x0000000000A4e21E2597DCac987455c48b12edBF",
-  }),
-  12341234: {
-    [OrderType.Dutch]: "0xbD7F9D0239f81C94b728d827a87b9864972661eC",
-    [OrderType.Dutch_V2]: "0x0000000000000000000000000000000000000000",
-    [OrderType.Relay]: "0x0000000000A4e21E2597DCac987455c48b12edBF",
+  11155111: {
+    [OrderType.Dutch_V2]: "0x0e22B6638161A89533940Db590E67A52474bEBcd",
   },
```

Comment:
not used anywhere as far as I can tell. We use REACTOR_ADDRESS_MAPPING instead

Reply:
yea no idea why we have both

Reply:
are you sure there are no external integrators that depend on this one? Maybe should alias it to be safe

Reply:
hmmm good call on aliasing. I did look through all our services as well as the frontend monorepo and found no reference of it. But going to make it an alias just to be 100% safe.